test/run

```diff
@@ -143,24 +161,30 @@ def print_counter(counter):
 
 
 def main():
+    args = get_args()
     logging.basicConfig(format="run-tests: %(message)s")
     test_dir = pathlib.Path(__file__).resolve().parent
     root = test_dir.parent
     litani = root / "litani"
+    if args.output_dir:
+        output_dir = pathlib.Path(args.output_dir)
+    else:
+        output_dir = test_dir / "output"
```

Comment:
Why don't you give `output_dir` a default value in `get_args()`?

Reply:
Agree